# Call service without inputs
---
bool success